examples/coordShiftTest/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.1)
+include(../../cmake/bob_robotics.cmake)
+BoB_project(SOURCES shift.cc
+            BOB_MODULES common
+	    THIRD_PARTY third_party_modules)
```

Comment:
You don't need the ``THIRD_PARTY`` array here. That's just for third-party modules which are git submodules or require extra dependencies etc.

Reply:
Also you don't want to commit your executables. I'd add them to a ``.gitignore`` file.